std_msgs/Float64MultiArray pos
std_msgs/Float64MultiArray quat
---
int8 success